changehc/params.json.template

```diff
@@ -4,6 +4,10 @@
   "cache_dir": "./cache",
   "input_denom_file": null,
   "input_covid_file": null,
+  "input_flu_file": null,
+  "input_mixed_file": null,
+  "input_flu_like_file": null,
+  "input_covid_like_file": null,
```

Comment:
This is borderline too many; prefer something like
```
"input": {
   "denom": null,
   "covid": null,
[...]
}
```

Reply:
Done